src/app/components/media-viewer-wrapper/media-viewer-wrapper.component.ts

```diff
@@ -27,22 +28,32 @@ export class MediaViewerWrapperComponent implements OnInit {
     public constructor(
         private windowService: WindowService,
         private featureToggleService: FeatureToggleService,
+        private sessionStorageService: SessionStorageService
     ) {
     }
 
     async ngOnInit() {
 
 
         const localStorageMedia = this.windowService.getLocalStorage(MEDIA_VIEWER);
-        if (localStorageMedia) {
+        let sessionStorageMedia = this.sessionStorageService.getItem(MEDIA_VIEWER);
+
+        if (!sessionStorageMedia && localStorageMedia) {
+            // move media viewer item from local to session storage to allow page refresh on multiple media viewer tabs
+            sessionStorageMedia = localStorageMedia;
```

Comment:
should we replace the setLocalStorage to setSessionStorage in ccd-case-ui-toolkit for 'media-viewer-info', and here change the getLocalStorage to getSessionStorage, rather than do an override sessionStorageMedia <= localStorageMedia here?

Reply:
Not quite sure what you're getting at here @rayliangatos. The local storage is set on the click event on the case documents tab of the case details page prior to loading a new tab. The file above deals with the new tab, and so needs to retrieve the value from local storage, move it to the session and then incorporate the results in the mv-media-viewer parameters which then display the media file. This allows multiple tabs to contain different files which don't get reset if the tab is refreshed.

Reply:
so do you need both localStorage and sessionStorage to accomplish this requirement? Or SessionStorage is enough? As I see in the code it's finally using the SessionStorage only.